src/Controller/ModulesController.php

```diff
@@ -79,6 +80,10 @@ public function index() : ?Response
         $this->set(compact('meta'));
         $this->set(compact('links'));
 
+        if (!empty($query['q'])) {
+            $this->set('searchtext', $query['q']);
```

Comment:
we could avoid passing `searchtext` to view and read from `$this->request->query` directly with a new helper method `Link::query('q')`

this will make it work for `trash` module too, where it's currently missing